packages/fxa-shared/metrics/amplitude.js

```diff
@@ -111,8 +111,8 @@ function mapSettingsEventProperties(...args) {
 }
 
 function mapSettingsVersion(eventType, eventCategory, eventTarget, data) {
-  if (data && data.settings_version) {
-    return { settings_version: data.settings_version };
+  if (data && data.settingsVersion) {
+    return { settings_version: data.settingsVersion };
```

Comment:
Updated to be consistent with camelCase in request payload but snake_case in Amplitude event properties.

Also, this already exists!  But the rest of the code required to get here was not.